tier1/checklist.md

```diff
@@ -0,0 +1,368 @@
+- [ ] **Facilitate Experimentation/Early Adoption**
+Working in the open encourages experimentation and early adoption of cutting-edge technologies,leading to faster innovation and improvemnet in software capabilities.
+
+### State3 the Risk(s) of Open Sourcing the Porject, if any
+
+- [ ] **Security Risks**
+Vulnerabilities may be exposed if the code is not thoroughly reviewed, potentially leading to security breaches or exploitation.(See: SECURITY.md)Does this project connect to any CMS-internal only systems? Does this project require authorization or authentication to operate? Does this project detail any non-public directories of CMS/HHS systems or people?
+- [ ] **Financial Risks**
+Costs may arise from maintaining code, community engagemnet, addressing security concerns, or subscription costs, hardware costs, specialized tooling or infrastructure costs, among others. Does this project require any ongoing financial costs or subscription fees?
+(e.g. - Cloud Hosting, Specialized build systems, paid maintainers, paid libraries or dependencies.)
+- [ ] **Privacy Risks**
+Does this project require access to non-public, non synthetic PII,PHI, or other internal-only CMS Systems containing such data or information?
+
+---
+
+### Questions
+
+- Does the code contain or touch any private information such as Personal Identifiable Information (PII) or Protected Health Information (PHI)?
+  - Can it be removed? Is it absolutely needed to function? Can it be shipped with synthetic data instead?
+ - Does the code interface with any CMS' internal-only systems(e.g. mainframes,JIRA instances, databases, etc..)?
+ - Does the repository contain any keys or credentials to access or authenticate with CMS' systems?
+   - Can it be removed or is it needed?
+
+   If you answered "yes" to any of the above  questions, your project may be 'sensitive' in nature, and require a more thorough review before sharing publicly. Please reach out to opensource@cms.hhs.gov for guidance. If you answer yes to any of these questions above, it is best to seek quidance **before** releasing open source.
+
+   **Results** 
+
+   *Insert Review Here*
+
+
+
+
+   ---
+
+
+
+
+   ### Code Review
+
+   The existing codebase should be given a one time, top-to-bottom code quality and security vulnerability review by two (or more) engineers who have written production code within the path two years, in the languages used in the project.Engineers should review credential management practices with the development team to ensure that any keys, passwords, or other sensitive configurations are not checked into source code or in the git history.
+
+   The engineers can be federal government employees or trusted partners from outside the agency from other contracts, or form independent testing contracts.Their names, organizations, comments and approval/disapproval on the overall codebase should be tracked in the documents.
```

Comment:
form -> from, add space between "contracts." and "Their", the documents -> this document